src/mon/OSDMonitor.cc

```diff
@@ -9582,7 +9617,7 @@ bool OSDMonitor::prepare_command_impl(MonOpRequestRef op,
       err = -EPERM;
       goto reply;
     } else if (!osdmap.exists(id)) {
-      ss << "osd." << id << " does not exist";
+      ss << "osd." << id << " does not exist.";
```

Comment:
please note this message is not sent to logging. so could you please split this change into another commit?